changelogs/unreleased/gh-10552-use-failover-priorities-when-selectin-bs-leader.md

```diff
@@ -0,0 +1,4 @@
+## feature/config
+
+- Now, when running in supervised failover mode the replicaset
+  with the highest priority is chosen by default (gh-10552).
```

Comment:
1. Should mention bootstrap somehow.
2. Replicaset -> instance.

Reply:
Thank you. Applied.